### PR TITLE
(#137) Address-0-0 Memo E2E Test

### LINKED ISSUE #137: Implement E2E tests for 0-0 memos
Implement e2e test for sending a bridge transfer with the option for user submission of ETH TX.
- Test that HBARs can be transferred
- Test that tokens can be transferred

### PR DESCRIPTION
**Pull request by georgiyazovaliiski**
**Detailed description**:
An E2E Test for a scenario where the memo is in a format Address-0-0. The validators should not be validating fee or sending an ETH Transaction Execution Message. I also modified the e2e configuration file and Hedera node initialization function a bit.

**Which issue(s) this PR fixes**:
Fixes #137 

**Checklist**
- [ ] Documentation added
- [x] Tests updated

### CODE REVIEW

**Review by failfmi — DISMISSED** [review]

**Review by failfmi — DISMISSED** [review]

File: e2e/e2e_test.go [hunk 1]
```diff
@@ -229,27 +240,21 @@ func verifyTopicMessages(setup *setup.Setup, transactionResponse hedera.Transact
 					ethTransMsgCollected++
 					fmt.Println(fmt.Sprintf("Received Ethereum Transaction Hash [%s]", msg.GetTopicEthTransactionMessage().EthTxHash))
 				}
-
-				// Check whether we collected everything
```
Comment (Daniel-K-Ivanov) [inline e2e/e2e_test.go:233]: I think that leaving the blocking channel is better. We will save execution time and not wait for 60 seconds timeout. In order to make it work we just have to remove the hardcoded 1 so the if becomes:
```go
if expectedValidatorsCount == ethSignaturesCollected &&
 ethTransMsgCollected == expectedEthTxMessageCount {
	c1 <- true
}
```

File: e2e/e2e_test.go [hunk 2]
```diff
@@ -188,14 +199,13 @@ func sendHbarsToBridgeAccount(setup *setup.Setup, memo string) (*hedera.Transact
 	return &res, err
 }
 
-func verifyTopicMessages(setup *setup.Setup, transactionResponse hedera.TransactionResponse, t *testing.T) string {
+func verifyTopicMessages(setup *setup.Setup, transactionResponse hedera.TransactionResponse, expectedSignaturesCount int, expectedEthTxMessageCount int, t *testing.T) string {
```
Comment (Daniel-K-Ivanov) [inline e2e/e2e_test.go:202]: Do we need to pass `expectedSignaturesCount`?

**Review by Daniel-K-Ivanov — CHANGES_REQUESTED** [review]
We should update the E2E docs with the network type as-well

File: e2e/e2e_test.go (hunk 2 above)
Comment (georgiyazovaliiski) [reply e2e/e2e_test.go:202]: This is needed in order to generalize the verification of different scenarios. With a format such as address-0-0 we are waiting for no ethTxMessage, therefore we pass a 0 at the assertion.

**Review by failfmi — APPROVED** [review]

**Review by Daniel-K-Ivanov — APPROVED** [review]

**Pull request merged**